sdk/servicebus/service-bus/src/util/parseUrl.browser.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export const parseURL = (rawUrl: string) => {
+  return new window.URL(rawUrl);
```

Comment:
```suggestion
  return new self.URL(rawUrl);
```

I think we should use `self` instead of `window` so that this also works if the user runs the code in a webworker.
https://developer.mozilla.org/en-US/docs/Web/API/Window/self

Reply:
Updated